website/README.md

```diff
@@ -0,0 +1,39 @@
+# x11libre-website
+
+A website for the X11Libre (A fork of [Xorg XServer](https://gitlab.freedesktop.org/xorg/xserver)) project by @metux and the contributors
+
+## About
+
+X11Libre is an open-source initiative focused on providing a modern, libre implementation of the X11 protocol and related tools.
+
+## Features
+
+- Open-source and community-driven
+- Modern web technologies
+- Documentation and resources for developers
+
+## Getting Started
+
+To contribute or deploy the website:
+
+```bash
+git clone https://github.com/metux/x11libre-website.git
```

Comment:
This repository does not exist, this PR is about merging these changes in the main repo, so what is the point of adding another repo?